script/setup

```diff
@@ -1,3 +1,5 @@
 #!/bin/sh
-
-rake db:setup db:migrate
+bundle install
+yarn
+bundle exec rake db:setup db:migrate
+NODE_OPTIONS=--openssl-legacy-provider bundle exec rake assets:precompile
```

Comment:
Not sure if these changes break something else / a dev routine when one runs everything on the host. Instead of expanding the script, one could also create a new one. For the docker stuff it is however quite handy to have all of these commands in one script. Should also be useful for "host developers".

Reply:
So this should go out in its own file?